flagging.md

```diff
@@ -71,6 +71,16 @@ If you'd like more information on the technical side of the system, drop into
 [Charcoal HQ](https://chat.stackexchange.com/rooms/11540) to have a chat about it.
 </section>
 
+<section>
+## What are spam waves? {#spam-waves}
+
+Spam waves are a tool that [metasmoke admins](https://charcoal-se.org/smokey/Privileges#admin-ms) have to set SmokeDetector to immediately cast any number of autoflags on detected posts matching pre-set criteria. That includes enough autoflags to immediately delete the post without requiring users to cast flags manually. This feature is separate from blacklisted/watched keywords - an admin must manually configure it in metasmoke.
```

Comment:
Please reflow the text; notice the maximum line length in the rest of the file and try to adhere to that. Markdown easily lets you whack newline in the middle of a sentence without it having any rendering semantics, and shorter lines make for less bewildering diffs down the line.

Reply:
@tripleee How's it look now? I've tried to match the max line length of the rest of the text